dotnet/Controllers/RoutesController.cs

```diff
@@ -327,6 +331,21 @@ public async Task<IActionResult> AddSearchDate()
             return Json("Done");
         }
 
+        public async Task<IActionResult> AddLocale()
```

Comment:
I'm not sure I understand why this task is necessary. Is it to populate the locale field on all the old reviews with an empty string? I would suggest leaving them as `null` instead, that way there's no need to modify all of the past records.